app/controllers/reports/instrument_day_reports_controller.rb

```diff
@@ -8,10 +8,10 @@ class InstrumentDayReportsController < ReportsController
 
     def self.reports
       @reports ||= HashWithIndifferentAccess.new(
-        reserved_quantity: -> (res) { Reports::InstrumentDayReport::ReservedQuantity.new(res) },
-        reserved_hours: -> (res) { Reports::InstrumentDayReport::ReservedHours.new(res) },
-        actual_quantity: -> (res) { Reports::InstrumentDayReport::ActualQuantity.new(res) },
-        actual_hours: -> (res) { Reports::InstrumentDayReport::ActualHours.new(res) },
+        reserved_quantity: ->(res) { Reports::InstrumentDayReport::ReservedQuantity.new(res) },
+        reserved_hours: ->(res) { Reports::InstrumentDayReport::ReservedHours.new(res) },
+        actual_quantity: ->(res) { Reports::InstrumentDayReport::ActualQuantity.new(res) },
+        actual_hours: ->(res) { Reports::InstrumentDayReport::ActualHours.new(res) },
       )
```

Comment:
I don't necessarily disagree, but we've definitely rubocopped this this way elsewhere recently, and I'd prefer to stay consistent.